spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaClientHttpConnector.java

```diff
@@ -48,7 +51,11 @@
     /**
      * Creates an {@link ArmeriaClientHttpConnector} with the default {@link ArmeriaClientConfigurator} and
      * {@link DataBufferFactoryWrapper}.
+     *
+     * @deprecated Use
+     *             {@link ArmeriaClientAutoConfiguration#clientHttpConnector(List, DataBufferFactoryWrapper)}.
```

Comment:
Should we add a static method to `ArmeriaClientAutoConfiguration` that takes no parameters?

Reply:
`ArmeriaClientAutoConfiguration` is a configuration bean. I don't think we need to offer another bean for `ClientHttpConnector` with `@Qualifier`. 🤔

Reply:
Ah, we don't. 😄